Source/Engine/UI/GUI/ContainerControl.cs

```diff
@@ -98,12 +97,19 @@ public bool IsLayoutLocked
         /// <summary>
         /// Gets or sets a value indicating whether apply clipping mask on children during rendering.
         /// </summary>
-        [EditorOrder(530), Tooltip("If checked, control will apply clipping mask on children during rendering.")]
-        public bool ClipChildren
-        {
-            get => _clipChildren;
-            set => _clipChildren = value;
-        }
+        [HideInEditor, Tooltip("If checked, control will apply clipping mask on children during rendering."), Obsolete("Use Clipping"), NoSerialize]
+        public bool ClipChildren;
+
+        /// <summary>
+        /// Controls how the clipping behavior of this Control.
+        /// Normally content that overflows the bounds of the Control continues rendering.
+        /// Enabling clipping prevents that overflowing content from being seen.
+        /// <para><b>Note:</b></para>
+        /// Controls in different clipping spaces can not be batched together, and so there is a performance cost to clipping.
+        /// Do not enable clipping unless a panel actually needs to prevent content from showing up outside its bounds.
+        /// </summary>
+        [EditorOrder(530), Tooltip("Controls how the clipping behavior of this Control.\nNormally content that overflows the bounds of the Control continues rendering.\nEnabling clipping prevents that overflowing content from being seen.\n\nNote:\nControls in different clipping spaces can not be batched together, and so there is a performance cost to clipping.\nDo not enable clipping unless a panel actually needs to prevent content from showing up outside its bounds.")]
```

Comment:
Maybe we should leave the original formatting and simply add new lines at the end of the original lines?

Reply:
u can't add new line string to a tooltip if u are asking about this
it just is not working

Reply:
I was talking about how the formatting used to look like: https://github.com/FlaxEngine/FlaxEngine/pull/2002/commits/437aa6cd094fdcb6ee1bc2b9e128b59fde0bab4a#diff-2e3b1425cc9e599c0dae72194347c78852075d8a8cff2eff1513fc0ea81db305R112

Reply:
the tool tip is not showing this type of formating because of +

Reply:
Ah, I see, okay then.